VTUL/dlp-access
# Adds How to document with associated images

Adds How to document with associated images

# What does this Pull Request do? (:star:)
This adds how to document with its associated image. 

# What's the changes? (:star:)
No changes to existing flow. 


# How should this be tested?

A description of what steps someone could take to:
* Make sure the links can be accessed. 
* Make sure the images attached can be accessed. 




# Interested parties
@goynejennifer
